File: PhysTasks/Task1.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main() {
    const double g = 9.81;
    
    double V, T;
    
    cout << "Введите начальную скорость (м/с): ";
    cin >> V;
    cout << "Введите время полета (с): ";
    cin >> T;

    double alpha = asin((g * T) / (2 * V)) * 180.0 / M_PI;

    cout << "Угол α: " << alpha << " градусов" << endl;

    return 0;
}

File: PhysTasks/Task1.py
import math

g = 9.81

V = float(input("Введите начальную скорость (м/с): "))
T = float(input("Введите время полета (с): "))

alpha = math.degrees(math.asin((g * T) / (2 * V)))

print(f"Угол α: {alpha:.2f} градусов")
